inst/workflow/1_mappings/STUDY.yaml

```diff
@@ -45,18 +45,45 @@ steps:
     params:
       data: Temp_CTMSStudyWide
       strGroupLevel: "Study"
+  - output: Temp_CTMSplanned
+    name: RunQuery
+    params:
+      df: Raw_STUDY
+      strQuery: "SELECT studyid as GroupID, num_plan_site as SiteTarget, num_plan_subj as ParticipantTarget FROM df"
   - output: Temp_StudyCountsWide
     name: RunQuery
     params:
       df: Mapped_SUBJ
       strQuery: "SELECT studyid as GroupID, COUNT(DISTINCT subjid) as ParticipantCount, COUNT(DISTINCT invid) as SiteCount FROM df GROUP BY studyid"
-  - output: Temp_StudyCounts
+  - output: Temp_CountTargetsWide
+    name: left_join
+    params:
+      x: Temp_CTMSplanned
+      "y": Temp_StudyCountsWide
+      by: GroupID
+  - output: Temp_CountTargetsWide
```

Comment:
maybe add _Site to the end of this output to avoid confusion with the output created in line 72

Reply:
I guess that was intentional because the function needs to be ran twice and I'm trying to append to the `Temp_CountTargetsWide` object and not necessarily create additional interim objects along the way, should I still proceed with the edit?

Reply:
ok, that makes sense then. we can keep it as is.

Reply:
My two cents - code is gernally easier to debug if you avoid re-assignment / overwriting.

Reply:
making naming changes for clarity